test/functional/apps/visualize/index.ts

```diff
@@ -50,7 +51,6 @@ export default function ({ getService, loadTestFile }: FtrProviderContext) {
       });
 
       // Test replaced vislib chart types
-      loadTestFile(require.resolve('./_area_chart'));
```

Comment:
Did we run this tests 2 times? I can see it in line 65

Reply:
we were yes

Reply:
not no morez! hahaaa